app/services/integrations/aggregator/payments/create_service.rb

```diff
@@ -44,7 +44,7 @@ def call
         def call_async
           return result.not_found_failure!(resource: 'payment') unless payment
 
-          ::Integrations::Aggregator::PAyments::CreateJob.perform_later(payment:)
+          ::Integrations::Aggregator::Payments::CreateJob.perform_later(payment:)
```

Comment:
Shouldn't we add this line in separate PR since this is actually calling integration. I would suggest having everything except this line so that we can merge this PR